tools/draw-importances.py

```diff
@@ -1,55 +1,180 @@
+                    top_left = (x * mv_block_size, y * mv_block_size)
+                    bottom_right = (top_left[0] + mv_block_size, top_left[1]
+                                                            + mv_block_size)
+                    draw.rectangle((top_left, bottom_right), fill=(int(imp /
+                                                    max_imp * 255), 0, 0, 128))
+
+        fig = plt.figure(figsize=(frame.width, frame.height), dpi=1)
+        fig.figimage(frame, cmap='gray')
+        plt.savefig(folder_path+'/'+re.search(rx, input[1]).group() + '.png',
+                                                    bbox_inches='tight')
+
+    else:
+        for (png_batch, bin_batch, imps_batch, rows_batch, cols_batch) in  zip(
+                   png_list, bin_list, imps_list, rows_list, col_list):
+            frame_batch = Image.open(png_batch)
+            frame_batch = frame_batch.resize((frame_batch.width *
+             frame_size_multiplier, frame_batch.height * frame_size_multiplier))
+            frame_batch = frame_batch.convert(mode='RGB')
+            draw = ImageDraw.Draw(frame_batch, mode='RGBA')
+
+            # Draw the grid.
+            for i in range(0, frame_batch.width, mv_block_size):
+                 draw.line(((i, 0), (i, frame_batch.height)), fill=(0, 0, 0, 255))
+            for i in range(0, frame_batch.height, mv_block_size):
+                draw.line(((0, i), (frame_batch.width, i)), fill=(0, 0, 0, 255))
+
+            # Draw the importances.
+            if max_imp > 0:
+                for y in range(rows_batch):
+                    for x in range(cols_batch):
+                        imp = imps_batch[y, x]
+                        top_left = (x * mv_block_size, y * mv_block_size)
+                        bottom_right = (top_left[0] + mv_block_size, top_left[1]
+                                                            + mv_block_size)
+                        draw.rectangle((top_left, bottom_right), fill=(int(imp
+                                                  / max_imp * 255), 0, 0, 128))
+
+            fig = plt.figure(figsize=(frame_batch.width, frame_batch.height),
+                                                                dpi=1)
+            fig.figimage(frame_batch, cmap='gray')
+            plt.savefig(folder_path + '/' + re.search(rx, bin_batch).group()
+                            + ".png", bbox_inches='tight')
+            if verbose:
+                click.secho(str(folder_path + '/'+
+                        re.search(rx, bin_batch).group() +".png"), fg="blue")
+
+
+    if raw:
+        click.secho("List of Figure Elements: "+str(fig_list))
+
+    if verbose:
+        if path == None:
+            click.secho("File name is "+ splitext(input[1])[0]+ ".png",
+                                                            fg="blue")
+        print("Folder Path: "+ os.getcwd()+"/"+ folder_path)
+
+    click.secho("File is sucessfully saved.", fg="green")
```

Comment:
Maybe "Processing done."? Since it can process multiple files.

Reply:
Sure :)